### PR TITLE
InsideUniversal ruleset

### PR DESCRIPTION
**Pull request by rambleraptor**
Ruleset for insideuniversal.net

### CODE REVIEW

File: src/chrome/content/rules/InsideUniversal.xml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<ruleset name="InsideUniversal">
+        <target host="forums.insideuniversal.net" />
+        <target host="www.insideuniversal.net" />
+        <target host="insideuniversal.net" />
+
```
Comment (YegorIevlev) [inline src/chrome/content/rules/InsideUniversal.xml]:
```xml
<securecookie host=".+" name=".+" />
```

Comment (rambleraptor) [reply src/chrome/content/rules/InsideUniversal.xml]: Change made. Thanks!

File: src/chrome/content/rules/InsideUniversal.xml [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+        <securecookie host=".+" name=".+" />
+
+        <rule from="^http:"
+                to="https:" />
```
Comment (YegorIevlev) [inline src/chrome/content/rules/InsideUniversal.xml]: Keep this on one line.

Comment (rambleraptor) [reply src/chrome/content/rules/InsideUniversal.xml]: Done.

**Comment by YegorIevlev** [comment]
@rambleraptor Wait for @J0WI or @Bisaloo to actually merge the PR.

**Comment by rambleraptor** [comment]
Fixed both of those

**Comment by J0WI** [comment]
Triggering Travis again.

**Comment by J0WI** [comment]
@rambleraptor it looks like they've blocked Travis. Could you add it to the [whitelist](https://github.com/EFForg/https-everywhere/blob/master/utils/ruleset-whitelist.csv)?

**Comment by rambleraptor** [comment]
Added to the whitelist!

File: utils/ruleset-whitelist.csv [hunk 3]
```diff
@@ -3776,3 +3776,4 @@ e0aa84f8ba58a339cabda74594b507c6427a9856a57b1ee3232c7f38ab80751a,1,1,1,Yimg.com.
 56e89e377a8cc26456aeaf380a689621afd18dd2e445399865874768f48943e1,0,1,1,Your-Mailing-List-Provider.xml
 ccf17e77a962a90d0134a342bbf02a2c48fceaa22672205030959e5eabbb54d1,1,1,1,YouVersion.xml
 95c9dd36a1b9d28f39272c9da9349eed53f7a3b8a1504e8f6729f67a0ab2a03a,0,1,1,yWorks.com.xml
+8f7914e006046e349b25fb919f40317c87fdd9ce249931917c24e48ab12247fb,1,1,1,InsideUniversal.xml
```
Comment (J0WI) [inline utils/ruleset-whitelist.csv]: `0,1,0` should be fine here

Comment (rambleraptor) [reply utils/ruleset-whitelist.csv]: Done!

Comment (J0WI) [reply utils/ruleset-whitelist.csv]: Thanks!

**Comment by rambleraptor** [comment]
Thank you! I know this PR took a lot of commenting + time to get in.

**Pull request merged**